templates/master/00-master/on-prem/files/haproxy.yaml

```diff
@@ -35,7 +35,7 @@ contents:
         - "-c"
         - |
           #/bin/bash
-          /host/bin/oc --kubeconfig /var/lib/kubelet/kubeconfig get nodes
+          /usr/bin/curl -o /dev/null -kLfs https://localhost:6443/readyz
```

Comment:
This doesn't do what we need. The point is to verify that we can get to the API via the api-int DNS record, so using localhost won't help. The address needs to be api-int.[cluster domain].

Reply:
thanks Ben.